commands.py

```diff
@@ -0,0 +1,18 @@
+import sublime
+import sublime_plugin
+from LSP.plugin.core.url import filename_to_uri
+
+class NewScalaFileCommand(sublime_plugin.TextCommand):
+
+    def run(self, edit: sublime.Edit, paths) -> None:
+        view = sublime.active_window().active_view()
```

Comment:
You already have the view available to you via `self.view`. There is no need to call `sublime.active_window().active_view()`.

Reply:
forgot that you can get one from the command